driver-core/src/test/functional/com/mongodb/internal/operation/ListDatabasesOperationSpecification.groovy

```diff
@@ -64,6 +64,14 @@ class ListDatabasesOperationSpecification extends OperationFunctionalSpecificati
         then:
         names.contains(getDatabaseName())
 
+        when:
```

Comment:
I imagine this test would pass even if we didn't add the field to the command?

Unfortunately testing this for real would be a headache, as we'd have to create a user without access to some database, and then assert that database doesn't appear in the results.  I don't think it's worth it.

Reply:
Sorry, I should have been more clear.  I think this test is better than nothing, since assuming the operation does include the field this ensures that the server doesn't blow up when the driver sends it.  So I think we should still have this test.

Reply:
Drat. Apologies for the misunderstanding. I have restored the test.